packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1792,7 +1792,7 @@ export default class Meeting extends StatelessWebexPlugin {
               this,
               {
                 file: 'meeting/index',
-                function: 'stopFloorRequest'
+                function: 'localShare'
```

Comment:
this in theory should be a function name, but the all the code around this area is not putting the function name, so I've put "localShare" here to be consistent with the surrounding code